### DISCUSSION TITLE
How to load the multiple files by the "init_data_sys" in input.json of dpgen?

### DISCUSSION THREAD

**Discussion by Guancred** [deepmodeling/dpgen#1411] [Q&A]
Hi, there are many folders for my initial data, whether there a simple method for me to load these folders ?  I have tried to set "*" in the dictionary of "init_data_sys" as following figure， but it does not work.
![image](https://github.com/deepmodeling/dpgen/assets/91767936/e912ed76-36ae-4021-904d-3f9bce4e3eed)

Thanks very much!
/

**Comment by njzjz** [comment]
A directory that contains all of the data can be given. The data is found resrecursively.
